domo-java-sdk-all/src/main/java/com/domo/sdk/datasets/DataSetClient.java

```diff
@@ -139,14 +139,19 @@ public void importData(String id, File contents) {
      * Export CSV data as an InputStream
      * @param id the DataSet id
      * @param includeHeader whether or not the CSV header row should be included
+     * @param disableFormulaInterpretation whether or not to use formula interpretation. When formula interpretation is disabled, data cells that could be interpreted as macros will be escaped with a '
      * @return a Java InputStream for data retrieval, be certain to close the input stream
      */
-    public InputStream exportData(String id, boolean includeHeader) {
-        HttpUrl url = urlBuilder.fromPathSegments(URL_BASE)
+    public InputStream exportData(String id, boolean includeHeader, Boolean disableFormulaInterpretation) {
```

Comment:
Is this a flag that will be used often externally?  This method is used by customers and by adding another parameter you will break their existing code, if they will almost always want this false, or true, we should create another method signature that matches the old and let them not worry about this new boolean.

Reply:
My concern was creating too many methods so the user didn't know which method to use.